src/main/java/decodes/decoder/ShefProcess.java

```diff
@@ -218,31 +218,30 @@ else if (c == '\n')
 				if (c == ':')
 				{
 					lineContext = LineContext.InComment;
-					debug(idx, "start of commnet");
-					continue;
+					debug(idx, "start of comment");
 				}
 				else if (Character.isWhitespace(c) || c == '/')
```

Comment:
This all looks reasonable but would probably be good to also include a simple test with the .A format to verify we aren't going to be adding a regression there.

Reply:
The header parsing for shef  (ShefPMParser.java) doesn't check for comment lines!

If I remove the comment lines from the sample .A data it appears to be happy.